frontend/src/scenes/session-recordings/player/rrweb/canvas/canvas-plugin.ts

```diff
@@ -215,7 +215,8 @@ export const CanvasReplayerPlugin = (events: eventWithTime[]): ReplayPlugin => {
 
             if (node.nodeName === 'CANVAS' && node.nodeType === 1) {
                 const el = containers.get(id) || document.createElement('img')
-                ;(node as HTMLCanvasElement).appendChild(el)
+                const parent = node.parentNode as Node
+                parent?.replaceChild?.(el, node as Node)
```

Comment:
are we risking losing attributes sizing the canvas here?
(just with a straight replace like this)
i guess... how do we know that the image has the correct dimensions?
not blocking... 

i'm just wondering how this _ever_ worked
and i've seen canvas recording work 🤷